Sprint-1/implement/max.js

```diff
@@ -1,4 +1,22 @@
 function findMax(elements) {
+    let max = -(Infinity);
+    if(elements.length == 0){
+        return Infinity
+    }
+    for(let i =0; i < elements.length ; i++){
+        
+        if(typeof elements[i] == "number" && elements[i] > max){
+            max = elements[i];
+        }
+    }
+    if(max == -(Infinity)){
+        return "there is no number in that array!!";
```

Comment:
There is also no numbers in an empty array, `[]`. Returning this value would somehow make the return value "not consistent".

Also, what do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
findMax([-Infinity]);
findMax([NaN]);
```